test/opentok-test.js

```diff
@@ -1389,6 +1431,59 @@ describe('#getBroadcast', function () {
   });
 });
 
+describe('#listBroadcasts', function () {
+  var opentok = new OpenTok('APIKEY', 'APISECRET');
+  var options = {
+    sessionId: 'SESSIONID',
+  };
+  afterEach(function () {
+    nock.cleanAll();
+  });
+  it('succeeds given valid parameters', function (done) {
+    mockListBroadcastsRequest();
+    opentok.listBroadcasts(function (err, broadcastList, totalCount) {
+      expect(err).to.be.null;
+      validateListBroadcastsObject(broadcastList);
+      validateTotalCount(totalCount);
+      done();
+    });
+  });
+
+  it('succeeds given options as valid parameters', function (done) {
+    mockListBroadcastsRequest({
+      sessionId: 'SESSIONID',
```

Comment:
I get a comma-dangle lint error here. There are also a number of other lint errors (some of which exist on the master branch, but they are easy fixes).

Reply:
I fixed the linter issues for the list broadcasts changes. Happy to do the others in a separate PR.